e2e/cli-e2e/tests/print-config.spec.ts

```diff
@@ -54,7 +54,13 @@ describe('print-config', () => {
         outputDir: join('tmp', 'ts'),
         filename: 'my-report',
       },
-      plugins: expect.any(Array),
+      plugins: expect.arrayContaining([
+        expect.objectContaining({ slug: 'eslint', title: 'ESLint' }),
+        expect.objectContaining({
+          slug: 'lighthouse',
+          title: 'ChromeDevTools Lighthouse',
+        }),
+      ]),
```

Comment:
I would move this more specific assertion to the `it.each` above, then it covers all formats. This test is just for testing CLI arg override, so the `plugins` assertion need not be so specific, because only `persist.filename` is really relevant to this scenario.